pyccel/parser/syntactic.py

```diff
@@ -59,9 +59,13 @@
 from pyccel.ast.core import PyccelPow, PyccelAdd, PyccelMul, PyccelDiv, PyccelMod, PyccelFloorDiv
 from pyccel.ast.core import PyccelEq,  PyccelNe,  PyccelLt,  PyccelLe,  PyccelGt,  PyccelGe
 from pyccel.ast.core import PyccelAnd, PyccelOr,  PyccelNot, PyccelMinus, PyccelAssociativeParenthesis
-from pyccel.ast.core  import PyccelOperator, PyccelUnary
+from pyccel.ast.core import PyccelOperator, PyccelUnary
 
-from pyccel.parser.utilities import fst_move_directives, preprocess_imports, preprocess_default_args
+from pyccel.ast.numbers import Complex
+
+from pyccel.ast.builtins import PythonComplex
```

Comment:
Unused import

Reply:
In the same file the following are also now unused:
```
parser/syntactic.py:5:0: W0611: Unused import traceback (unused-import)
parser/syntactic.py:13:0: W0611: Unused BinaryOperatorNode imported from redbaron (unused-import)
parser/syntactic.py:22:0: W0611: Unused AssociativeParenthesisNode imported from redbaron (unused-import)
parser/syntactic.py:54:0: W0611: Unused Dlist imported from pyccel.ast (unused-import)
parser/syntactic.py:66:0: W0611: Unused PythonComplex imported from pyccel.ast.builtins (unused-import)
```

Reply:
they were never properly working, but now they are broken completely

Reply:
Is this comment on the wrong thread?